RecoTracker/LST/src/alpaka/ES_ModulesDev.cc

```diff
@@ -0,0 +1,10 @@
+#include "RecoTracker/LSTCore/interface/alpaka/LST.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/typelookup.h"
+
+// Temporary hack: The DevHost instantiation is needed in both CPU and GPU plugins,
+// whereas the (non-host-)Device instantiation only in the GPU plugin
+TYPELOOKUP_DATA_REG(SDL::LSTESHostData<SDL::Dev>);
+TYPELOOKUP_DATA_REG(SDL::LSTESDeviceData<SDL::DevHost>);
+TYPELOOKUP_DATA_REG(ALPAKA_ACCELERATOR_NAMESPACE::ESDeviceProduct<std::unique_ptr<SDL::LSTESHostData<SDL::Dev>>>);
```

Comment:
What is the intention of this line?

Reply:
> What is the intention of this line?

@ariostas can give more detail.
IIUC, the code did not link otherwise.

Reply:
Yeah, without this line it fails to link. It's probably that something about the CopyAsync wasn't implemented right. I'll describe it more in the other comment below.